src/ORM/Connect/MySQLSchemaManager.php

```diff
@@ -241,6 +241,23 @@ public function renameField($tableName, $oldName, $newName)
 
     protected static $_cache_collation_info = [];
 
+    public function shouldUseIntegerWidth()
```

Comment:
Ideally we would have some unit test covering. It might be possible to do it with some semi-elaborate mock.

It's probably good enough as it is.

Reply:
Done